### DISCUSSION TITLE
【揭秘】法厄同们的突然失踪？音讯全无背后竟然隐藏着.........

### DISCUSSION THREAD

**Discussion by sterve521** [share121/inter-knot#977] [其他]
![1723290442036](https://github.com/user-attachments/assets/37e2188e-fbce-497c-8350-aab01d3fd8ff)
独家消息！独家消息！原因居然是因为学校违规开学导致传奇绳匠法厄同无法工作，发帖表示大家不用担心，一周后法厄同必将传奇归来
![Cache_-174aba50f26e93c9](https://github.com/user-attachments/assets/6ff44c68-89b6-43f8-b6ae-6328ec075916)

**Comment by chengmmmmmm** [comment]
好家伙fufu

**Comment by sterve521** [comment]
重点是fufu吗？悲~
